.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,65 @@
+name: Feature request
+description: Suggest a new feature for MSAL .NET.
```

Comment:
```suggestion
Suggest a new feature for MSAL.NET.
```

Reply:
we need that `description:` field, added it back